### PR TITLE
#3390 Limiting of characters

### LINKED ISSUE #3390: Sensor name and code need validation
## Describe the bug

When adding a new sensor, the sensor name and code should be validated. Same in the edit page.

### To reproduce

1. Go to a new sensor or edit an old one
2. Change the name/code
3. theres no validation

### Expected behaviour

Think we limit code to < 10 chars, and name to <50

### Proposed Solution

Just a simple length check

### Version and device info

- App version:
- Tablet model:
- OS version:

### Additional context

N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3390 

## Change summary

- Adds simple length check on sensor code/name input

## Testing

- [ ] can't enter a sensor name greater than 50 characters
- [ ] can't enter a sensor code greater than 50 characters

### Related areas to think about

Be good to have validation feedback for the user one day ;-)

### CODE REVIEW

File: src/reducers/SensorDetailReducer.js [hunk 1]
```diff
@@ -32,12 +32,18 @@ export const SensorDetailReducer = (state = initialState(), action) => {
...
+      const isValid = name.length < 50;
+      if (!isValid) return state;
+
       return { ...state, name };
     }
     case SENSOR_DETAIL_ACTIONS.UPDATE_CODE: {
       const { payload } = action;
       const { code } = payload;
 
+      const isValid = code.length < 50;
```
Comment (katherine-sussol) [inline src/reducers/SensorDetailReducer.js:44]: Should this be 10 as well? (unless we're intentionally being more restrictive when creating?)

**Review by katherine-sussol — COMMENTED** [review]
Looks good, just one question before hitting approve 😸 .

File: src/reducers/SensorDetailReducer.js (hunk 1 above)
Comment (joshxg) [reply src/reducers/SensorDetailReducer.js]: Whoopsies!


4c52af3

🙈

**Review by katherine-sussol — APPROVED** [review]
👍

**Pull request merged**